### ISSUE TITLE
emojicker reveal

### ISSUE THREAD

**Issue by Celestemysterios** [vendetta-mod/ideas#207]
### Discord username

celestemyst 

### Plugin description

so basically if a emoji/sticker does not show the server it is from (it says smth like private server) it reveals the server it came from
so like if the emoji/sticker is from a private server the plugin would reveal the server the emoji/sticker came from

### Elaborate

i feel like i explained above so ima say how you pronouce emojicker
e-mo-j-ick-er
so basically start saying emoji but stop at the i and instead say icker
emojicker

### ​

- [X] I have verified that this request is indeed original

**Comment by wingio** [comment]
Not possible
